index.html

```diff
@@ -319,30 +277,27 @@ <h1 class="text-sm flex justify-center p-5 dark:bg-ys-black">
                   </button>
                 </div>
                 <div class="p-8 dark:bg-ys-backgroundAndText">
-                  <div id="code-wrapper" data-cy="util:code-editor">
-                    <textarea
-                      class="dark:bg-ys-backgroundAndText bg-ys-amethyst-200 p-5 outline-none"
-                      name=""
-                      id=""
-                      aria-label=""
-                      rows="6"
-                    >
-                    </textarea>
+                  <div>
+                    <div
+                      class="code-editor"
+                      id="code-editor"
+                      aria-label="Code editor"
+                    ></div>
                     <div class="flex gap-1 pt-5">
-                      <input
-                        type="text"
-                        class="w-20 text-center dark:bg-ys-overlay-15"
-                        placeholder="cell"
-                        aria-label=""
-                        id=""
-                      />
+                      <div>
+                        <button
+                          type="text"
+                          class="w-20 text-center dark:bg-ys-overlay-15 text-white"
+                          id="selected-cell"
+                        ></button>
+                      </div>
```

Comment:
Should this be a button? Is this supposed to be clickable? Or just show the value in a styled div?

Reply:
I see. Sounds good😊